libqtile/notify.py

```diff
@@ -22,63 +23,55 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-"""
-    If dbus is available, this module implements a
-    org.freedesktop.Notifications service.
-"""
-from libqtile.log_utils import logger
+# type: ignore
+
+from dbus_next.aio import MessageBus  # noqa: I001
+from dbus_next.service import ServiceInterface, method, signal  # noqa: I001
 
-try:
-    import dbus
-    from dbus import service
-    from dbus.mainloop.glib import DBusGMainLoop
-    DBusGMainLoop(set_as_default=True)
-    has_dbus = True
-except ImportError:
-    has_dbus = False
```

Comment:
I think we should keep dbus as an optional dependency, and thus keep that kind of stuff. That would also resolve your problem with package maintainers. :)

Reply:
Forget my comment, Tycho is right, a pure-python dependency is not a big deal.

Reply:
I think that's right. 

My question is more about etiquette/procedure. I assume it's the package maintainers' duty to identify dependency changes but wanted to check.

Reply:
I guess we'll see if people complain or not. :)